model_mommy/mommy.py

```diff
@@ -381,7 +381,12 @@ def _skip_field(self, field):
     def _handle_one_to_many(self, instance, attrs):
         for k, v in attrs.items():
             manager = getattr(instance, k)
-            manager.set(v, clear=True)
+
+            try:
+                manager.set(v, bulk=False, clear=True)
```

Comment:
I am not exactly sure how do we get here `TypeError` for many-to-many if this method is meant to be `_handle_one_to_many `.

It seems to me that we need to revisit added test to be more precise (see comment to the test).